### PR TITLE
(feat taxprofiler): Store multiqc intermediate json files

### PR DESCRIPTION
**Pull request by sofstam**
## Added

* MultiQC intermediate files to HK

### How to prepare for test
- [ ] ssh to hasta.scilifelab.se
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for hasta):
`bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_hermes -t hermes -b taxprofiler_intermediate_json_files`

### How to test
- [ ] `housekeeper get bundle -v propersheep`

Screenshot 2024-04-15 at 15 12 13

### Expected test outcome
- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review
- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan
- [ ] Document in ...
- [ ] Deploy this branch
- [ ] Inform to ...

### CODE REVIEW

**Review by fevac — APPROVED** [review]
nice! 🌟

**Review by ivadym — APPROVED** [review]
Nice ⭐ 

Two things:
- Could you also update [this excel sheet](https://docs.google.com/spreadsheets/d/1OS5UVIFcSVdcFNCAbJZehUn15OB6_UtVTr6FSaccSkU/edit?pli=1#gid=0) so it's easier for @ChrOertlin to implement the changes in Janus? 
- We've added the [tags as constants](https://github.com/Clinical-Genomics/hermes/pull/103), so I would recommend using them in the configs from now on. Would be great if you could make use of them instead of keep hardcoding strings 🙏

**Comment by sofstam** [comment]
Thanks for the comments. 

* Updated the excel document
* Saw this now that the tests are failing, updating the PR :)

**Comment by sofstam** [comment]
PR updated!

File: cg_hermes/config/taxprofiler.py [hunk 1]
```diff
@@ -1,90 +1,97 @@
...
+    BioinfoToolsTags,
+)
 
 TAXPROFILER_COMMON_TAGS = {
     frozenset({"kraken2_combined_report", "kraken2"}): {
         "is_mandatory": True,
         "bundle_id": True,
-        "tags": ["kraken2", "combined-report"],
+        "tags": [TaxprofilerTags.KRAKEN2, TaxprofilerTags.COMBINED_REPORT],
         "used_by": ["clinical-delivery", "long-term-storage"],
```
Comment (ivadym) [inline cg_hermes/config/taxprofiler.py:17]: We also have constants for the used by fields 🙏

**Review by ivadym — APPROVED** [review]
Looks great with the constants ⭐

File: cg_hermes/config/taxprofiler.py (hunk 1 above)
Comment (sofstam) [reply cg_hermes/config/taxprofiler.py]: Updated the PR!

Comment (sofstam) [reply cg_hermes/config/taxprofiler.py]: Updated the description after testing, I will merge.

**Comment by sofstam** [comment]
Deploy on stage:
Screenshot 2024-04-15 at 15 24 35

**Comment by sofstam** [comment]
Deploy on production:

Screenshot 2024-04-15 at 15 27 43

**Pull request merged**